index.html

```diff
@@ -323,24 +334,77 @@ <h3 class="custom-tags-font default-custom-font about-tag-color">Selenium</h3>
           <h2 class="about-list-text project-title-custom-font">Skills</h2>
           <ul class="about-tags flex-container-row">
             <li class="about-tag">
-                <h3 class="custom-tags-font default-custom-font about-tag-color">Codekit</h3>
+              <h3 class="custom-tags-font default-custom-font about-tag-color">Codekit</h3>
             </li>
             <li class="about-tag">
-                <h3 class="custom-tags-font default-custom-font about-tag-color">Github</h3>
+              <h3 class="custom-tags-font default-custom-font about-tag-color">Github</h3>
             </li>
             <li class="about-tag">
-                <h3 class="custom-tags-font default-custom-font about-tag-color">Codepen</h3> 
+              <h3 class="custom-tags-font default-custom-font about-tag-color">Codepen</h3>
             </li>
             <li class="about-tag">
-                <h3 class="custom-tags-font default-custom-font about-tag-color">Gitlab</h3>
+              <h3 class="custom-tags-font default-custom-font about-tag-color">Gitlab</h3>
             </li>
             <li class="about-tag">
               <h3 class="custom-tags-font default-custom-font about-tag-color">Terminal</h3>
-          </li>
+            </li>
           </ul>
         </li>
       </ul>
     </section>
-    
-  </body>
+    <!-- Add by charles Andrew- code Partner  -->
+    <!-- conatct form section start  -->
+    <section id="contact" class="section-contact-container flex-container-column">
+      <h2 class="legend-font-desktop primary-text-custom-font">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
```

Comment:
- [ ] Kindly remove this text as it's existed twice to match the Figma design.
![image](https://user-images.githubusercontent.com/24703171/228565276-a5f43ba5-1663-43e1-b5fa-e41b2dbbbd96.png)